project-resources/requesting_LFIT_support.md

```diff
@@ -1,45 +1,43 @@
-# Requesting Linux Foundation IT Support 
+# Requesting Linux Foundation IT Support Services 
 
-This guide outlines the procedures and best practices for requesting assistance from the Linux Foundation IT Operations team.
+## Introduction
+This guide is intended for OpenJS projects that have formally engaged with the Linux Foundation IT team to support one or more of their services. It outlines the available IT support services, how to submit a request and the process for escalating issues.
 
-## Who Is This Document For?
+## Services Available
```

Comment:
I think it would be helpful to have a list of the SASS products that are supported/ that we have service agreements with e.g. cloudflare, github

Reply:
Can you submit a patch with the agreements OpenJS has?